.github/workflows/sycl_precommit_aws.yml

```diff
@@ -1,114 +0,0 @@
-name: E2E on AWS CUDA
-run-name: E2E on AWS CUDA - ${{ github.event.workflow_run.display_title }}
-# We have to keep pre-commit AWS CUDA testing in a separate workflow because we
-# need access to AWS secret and that isn't available on pull_request jobs for
-# PRs from forks. And github's "require approval for all outside collaborators"
-# is bypassed on pull_request_target.
-#
-# Also, we use commit status and not check suite/run (which, in theory, is more
-# powerful) due to https://github.com/orgs/community/discussions/24616.
-
-on:
-  workflow_run:
-    workflows: [SYCL Pre Commit on Linux]
-    types:
-      - completed
-    branches-ignore:
-      - sycl-mlir
```

Comment:
I thought adding these would be enough. e.g., https://github.com/intel/llvm/pull/10790 doesn't run the CUDA job.

Reply:
@aelovikov-intel Would adding the lines below in `sycl` branch work?
```
    branches-ignore:
      - sycl-mlir
```

Reply:
Where did you get those `branches`/`branches-ignore` for `workflow_run` trigger? I don't see anything in https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run. As I said earlier in MS Teams, I think MLIR should just create their own workflow for precommit, separate from `sycl_precommit_linux.yml`. Another benefit of that would be that it would be possible to select pre-commit tasks for a single project (unlike now).

Reply:
> Where did you get those branches/branches-ignore for workflow_run trigger?

https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-branches-for-workflow-run-events

> I think MLIR should just create their own workflow for precommit

Can you please elaborate on how to do that? I cannot find an invocation point to `sycl_precommit_aws.yml` or `sycl_precommit_linux.yml`.

Reply:
Done. I think this makes sense now

Reply:
Please update PR title and description.